src/main/java/org/bigml/binding/BigMLClient.java

```diff
@@ -199,34 +199,20 @@ public static void resetInstance() {
         instance = null;
     }
 
+    private String setting(final String setting) {
+      String value = System.getProperty(setting);
+      if (value == null)
+        value = System.getenv(setting);
+      return value;
+    }
+
     /**
      * Initialization object.
      */
     private void init(final boolean devMode) throws AuthenticationException {
         this.devMode = devMode;
         initConfiguration();
-
-        this.bigmlUser = System.getProperty("BIGML_USERNAME");
-        this.bigmlApiKey = System.getProperty("BIGML_API_KEY");
-        if (this.bigmlUser == null || this.bigmlUser.equals("")
-                || this.bigmlApiKey == null || this.bigmlApiKey.equals("")) {
-            this.bigmlUser = props.getProperty("BIGML_USERNAME");
-            this.bigmlApiKey = props.getProperty("BIGML_API_KEY");
-            if (this.bigmlUser == null || this.bigmlUser.equals("")
-                    || this.bigmlApiKey == null || this.bigmlApiKey.equals("")) {
-                AuthenticationException ex = new AuthenticationException(
-                        "Missing authentication information.");
-                logger.info(instance.toString(), ex);
-                throw ex;
-            }
-        }
-
-        // The seed to be used to create deterministic samples and models
```

Comment:
I know that's a deeper change, but the seed is not "connection" property. Is one parameter in the models configuration... and sometimes there's more than one seed, so I don't think this should be set here. We can leave that for another PR but then I'll add an issue...